ai-service/src/ai_service/main.py

```
@@ -1,112 +1,176 @@
 
+# FastAPI imports
+from fastapi import FastAPI, Request
+from pydantic import BaseModel
+import uvicorn
+
+app = FastAPI()
+
+
+# Most elegant solution - FastAPI exception handlers
+@app.exception_handler(errors.AIServiceError)
+async def ai_service_error_handler(request: Request, exc: errors.AIServiceError):
```

Ruff (0.12.2)

Missing return type annotation for public function `ai_service_error_handler`

(ANN201)